styles.css

```diff
@@ -40,6 +40,60 @@ a:hover {
     align-items: center;
   }
 
+  .hidden-toolbar {
+    display: none;
+  }
+
+  /* Hamburger clicked */
+
+  .not-cliked {
+    display: none;
+  }
+
+  .clicked {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-image: url('images/header-shapes-mobile.png');
+    background-position: right top;
+    background-repeat: no-repeat;
+    height: 100vh;
+  }
```

Comment:
- Hey Team, please, let's improve our design of the mobile menu to match the design as much as we can. See the images below:

   - Let's use the correct color on the text and cancel button
   - Use the correct background on the menu container and add the background blur
   - Align the text content correctly
   - The container should take up the whole height of the screen



*Figma design*
![Screenshot from 2023-03-06 20-24-22](https://user-images.githubusercontent.com/59397159/223211001-c4b2ca68-a871-4c49-b9f2-7ecf79429201.png)

*Your design*
![Screenshot from 2023-03-06 20-23-08](https://user-images.githubusercontent.com/59397159/223211007-0ff3d1b2-b769-4ba9-9b89-993b73d7d1e4.png)